.gitignore

```diff
@@ -11,3 +11,5 @@ coverage.out
 tags
 temp_parser_file
 y.output
+
+cscope.*
```

Comment:
Why not put this rule into your global .gitignore file, and remove it from this file?